how_to/join_leadership_council.md

```diff
@@ -1 +1,9 @@
 # How to Join the Leadership Council
+
+To join the [Leadership Council](https://github.com/BlackPythonDevs/blackpythondevs/blob/main/roles/leadership-councilmembers.md), you must satisfy the following requirements:
+
+* You must be a recognized leader in any local or online Python community.
+* You must be a member of Black Python Devs.
+* You must be nominated by at least one leader of the Black Python Devs community.
+* You must be voted for by a simple majority of the existing [Leadership Council](https://github.com/BlackPythonDevs/blackpythondevs/blob/main/roles/leadership-councilmembers.md) members.
```

Comment:
This is technically not correct.

```suggestion
* Your joining must not be blocked by an existing [Leadership Council](https://github.com/BlackPythonDevs/blackpythondevs/blob/main/roles/leadership-councilmembers.md) member.
```

This distinction is that you are added after a 48-hour waiting period where any existing member can raise a concern about the members joining. After a discussion or if there is no concern raised the member is added to the Leadership Council.